index.js

```diff
@@ -65,8 +65,8 @@ exports.getArrayPages = function(req) {
     if (typeof pageCount !== 'number' || pageCount < 0)
       throw new Error('express-paginate: `pageCount` is not a number >= 0');
 
-    currentPage = Number.parseInt(currentPage);
-    if(Number.isNaN(currentPage) || currentPage < 0)
+    currentPage = Number.parseInt(currentPage, 10);
```

Comment:
Can you make this line and line 31 just have `parseInt` instead of `Number.parseInt`?  You don't need to use `Number.` prefix.  The lines should just look like:

```js
currentPage = parseInt(currentPage, 10);
```

and

```js
query.page = parseInt(query.page, 10);
```

Other than that we're good!